pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -931,4 +959,8 @@ public boolean isQueryRateLimitDisabled() {
   private String constructTableConfigPath(String tableNameWithType) {
     return "/CONFIGS/TABLE/" + tableNameWithType;
   }
+
+  private String constructLogicalTableConfigPath(String tableName) {
+    return "/LOGICAL/TABLE/" + tableName;
```

Comment:
I remember there is a constant for the literal. Can that be used instead ?

Reply:
Actually, there are no common constants defined for these (various ZK paths). For most part they are defined locally wherever they are used.